contracts/andromeda-portal-ado/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
This whole thing looks very similar to https://github.com/andromedaprotocol/andromeda-contracts/tree/development/contracts/andromeda_ics20

Is this just extended version of it?